### DISCUSSION TITLE
Unable to produce pdf report

### DISCUSSION THREAD

**Discussion by alinciprian** [Cyfrin/security-and-auditing-full-course-s23#32] [Q&A]
When running the command `pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings` I get the following error :
`! LaTeX Error: File footnotebackref.sty' not found`

**Comment by alinciprian** [comment]
brew install mactex

**Comment by 0zKillua** [comment]
I'm having similar issue. Since MacTex is larger in size, I installed basicTex, found on mactex page.(https://www.tug.org/mactex/morepackages.html) . what can i probably do?

my error: 

Error producing PDF.
! LaTeX Error: File `footnotebackref.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name: 
! Emergency stop.
<read *> 
         
l.109 \setlength
